pkg/consolegraphql/resolvers/resolver_address.go

```diff
@@ -187,5 +187,15 @@ func (r *mutationResolver) PurgeAddress(ctx context.Context, input metav1.Object
 }
 
 func (r *queryResolver) AddressCommand(ctx context.Context, input v1beta1.Address) (string, error) {
-	return "oc apply blah", nil
+
+	return `apiVersion: enmasse.io/v1beta1
```

Comment:
I thought the intent was to provide a complete UNIX shell command, which could be copied and pasted straight to the terminal?  I was expecting a here document.

oc apply -f << EOF
...
EOF

Reply:
@k-wall that was what I was expecting to do.  But then I checked the wireframes and saw.  https://marvelapp.com/i1igj54/screen/58093530 .  And ended up just providing the yaml.
I do agree that being able to cut and paste would be more useful.  I can quickly add the commands.